packaging/debian/datadog-agent.init

```diff
@@ -151,6 +152,11 @@ case "$1" in
         exit $?
         ;;
 
+    reload)
+        kill -12 `cat $COLLECTOR_PIDFILE`
```

Comment:
Watchout SIGUSR2 is not always 12, this is platform dependant. You should use `kill -SIGUSR2 instead`